lib/codegen/meta/cdsl/instructions.py

```diff
@@ -209,7 +209,7 @@ def _verify_polymorphic(self):
                     self.other_typevars = self._verify_ctrl_typevar(tv)
                     self.ctrl_typevar = tv
                     self.use_typevar_operand = True
-            except RuntimeError as e:
+            except Exception as e:
```

Comment:
Is there some new exception being raised that needs this change?

Reply:
I need to fix this differently; thank you for pointing it out. I took a shortcut here and forgot to go back.

What's going on is this:
```
  File "/Users/tyler/.devly/library/repos/isolation/lib/cretonne/lib/codegen/meta/cdsl/instructions.py", line 209, in _verify_polymorphic
    opnum = self.value_opnums[self.format.typevar_operand]
IndexError: tuple index out of range
```
What is happening is that, unlike every other polymorphic instruction (I think), `load_complex` has no "value opnums". All the values are in a `ValueList`. So, when we index the empty `value_opnums` with the `typevar_operand` in the format, which is set to 0, we get an `IndexError`.

As I look at this again, I think it's probably a bug somewhere in the `InstructionFormat` class. I'll poke at that and try to figure out what the problem is.